modules/gnetBidAdapter.js

```diff
@@ -0,0 +1,111 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import * as utils from '../src/utils.js';
+import { BANNER } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'gnet';
+const ENDPOINT = 'https://grumft.cbykmedia.com/prebid/prebid.php';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
```

Comment:
Does gnet have a registered GVL ID?

If so, it is recommended to add it to your spec so it functions accordingly in GDPR scenarios.

See https://github.com/prebid/Prebid.js/blob/master/modules/rubiconBidAdapter.js#L114 as an example

Reply:
No, we don't have a GVL ID yet